### PR TITLE
TLC-109: Fix Labels

### PR DESCRIPTION
**Pull request by ouiliam**
modified:   configuration/messageproperties/impl_en.properties
	modified:   configuration/personattributetypes/personAttributeTypes.csv

### CODE REVIEW

File: configuration/personattributetypes/personAttributeTypes.csv [hunk 1]
```diff
@@ -6,9 +6,8 @@ c3bcd296-eb82-4451-a4b4-eedd1d5f322f,,impl.phoneNumber,impl.phoneNumber.desc,jav
 1df3e8c6-564e-4da9-9094-5d36f68100c9,,impl.distanceFromHealthCenter,impl.distanceFromHealthCenter.desc,org.openmrs.Concept,acf4c89a-1731-430a-aefd-a5db1fe96ee8,,,
 a4e9107d-175b-4e4d-b992-3ba155b94e4b,,impl.patientStatus,impl.patientStatus.desc,org.openmrs.Concept,cede5394-4d47-446f-85cc-e649673af6ea,,,
 f155a740-c252-4e45-bea6-51fd9706dcf7,,impl.other,impl.other.desc,java.lang.String,,,,
-82325788-3f10-11e4-adec-0800271c1b75,,givenNameLocal,Khmer_Given_Name,java.lang.String,,TRUE,,
-8233a58a-3f10-11e4-adec-0800271c1b75,,familyNameLocal,Khmer_Family_Name,java.lang.String,,TRUE,,
-8234e7b0-3f10-11e4-adec-0800271c1b75,,middleNameLocal,Khmer_Middle_Names,java.lang.String,,,,
```
Comment (rbuisson) [inline configuration/personattributetypes/personAttributeTypes.csv:11]: @ouiliam this won't delete the existing *middleNameLocal* from a database where it has been created previously. If the intent is to do this, the you should keep the line, and enter `TRUE` in the *Void/Retire* column.

File: configuration/messageproperties/impl_en.properties [hunk 2]
```diff
@@ -14,9 +14,6 @@ impl.nationality=Nationality
 impl.nationality.desc=Nationality
 impl.email=Email
 impl.email.desc=Primary email address
-Khmer_Given_Name=Name in Khmer
```
Comment (rbuisson) [inline configuration/messageproperties/impl_en.properties:17]: Those are backend messages and I think are needed. I don't recall exactly why but I am pretty sure that we have done that in purpose. Unless you specifically think that this generates a bug somehow, I suggest we keep those messages.

File: configuration/personattributetypes/personAttributeTypes.csv (hunk 1 above)
Comment (rbuisson) [reply configuration/personattributetypes/personAttributeTypes.csv:11]: Furthermore this will break the i18n because we hard code the label in English.

Comment (ouiliam) [reply configuration/personattributetypes/personAttributeTypes.csv:11]: this fixed search results labels in offline, but it's not a high priority now so don't need to merge

**Pull request closed without merging**